src/lib/y2s390/dialogs/format_dialog.rb

```diff
@@ -43,6 +43,8 @@ class FormatDialog < ::UI::Dialog
       #
       # @param dasds [Array<Y2S390::Dasd>] list of DASDs to be formatted
       def initialize(dasds)
+        super()
```

Comment:
Are we sure this change is safe? It's actually the first real change I have seen (with implications that may go beyond style and readability).

Reply:
yes, change is safe and I do not many times for newer rubocop. Basically when you have class that is child of another, rubocop wants you to call its initialize to ensure that it is properly initialized. And I need to use `super()` as `super` will pass its arguments and UI::Dialog constructor is without params.
For more such changes see e.g. https://github.com/yast/yast-storage-ng/pull/1278/commits/e76cfb2ef98c18753c29558ee808f1aae2293ad3